src/tailwind.css

```diff
@@ -1,4 +1,29 @@
 @import url("https://use.fontawesome.com/releases/v5.2.0/css/all.css");
+@import url("//hello.myfonts.net/count/481147");
```

Comment:
just checking, what is this url arh?

Reply:
i think is to register font load count to the myfonts provider, part of the installation instructions provided. but apparently u can do without too